starting_from_scratch_lab/lab 37/lab_37_tar3.js

```diff
@@ -0,0 +1,50 @@
+let game = [
+  ["X", "O", "O"],
+  ["O", "X", "O"],
+  ["O", "O", "X"],
+]; // diag
+
+let game = [
+    ["X", "O", "O"],
+    ["X", "O", "O"],
+    ["O", "O", "X"],
+  ]; // colm
+
+  let game = [
+    ["X", "X", "X"],
+    ["O", "X", "O"],
+    ["O", "O", "X"],
+  ]; // row
+
```

Comment:
Please don't use multiple "let" calls with the same name in the same scope (it doesn't work everywhere and considered a mistake).
Either create an array or 3 different variables